### ISSUE TITLE
[Accessibility: Resize text] Unable to see all template information when adding guest with zoom 200%

### ISSUE THREAD

**Issue by alinefm** [kimchi-project/kimchi#896]
![4f14651d42c51708](https://cloud.githubusercontent.com/assets/4921718/14019937/b79b4d0a-f1b3-11e5-80ee-684e4480b36b.jpg)
